package.json

```diff
@@ -54,6 +54,7 @@
     "underscore-template-loader": "^0.7.3",
     "webpack": "^1.13.2",
     "webpack-dev-server": "^1.15.2",
+    "webpack-dotenv-plugin": "^1.4.0",
```

Comment:
Why are we adding this? We already have a system in place for reading environment variables and loading the end files.